app/client/tsconfig.json

```
@@ -16,7 +16,8 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "downlevelIteration": true
```

```suggestion
    "baseUrl": "src"
```

I think we can remove the `downlevelIteration` flag and change the target on line 3 to `es6`. All modern browsers support es6 now. 

I checked our `browserlist` configuration (in package.json) and pretty much all of the browsers in that config support es6. Safari 9.1 is the one outlier, which only has partial support for es6. I tested Safari 9.1, with browserstack, and it doesn't even work if we include the downlevelIteration flag and set the target to es5. 

I also looked at HMW and over there we only support Safari 14 or greater. That is probably due to Esri though. 

I'm not sure if you have used this or not, but you can use [browsersl.ist](https://browsersl.ist/#q=%3E0.2%25+and+not+dead+and+not+op_mini+all) to see what browsers will come back from our browserlist config.

---

Sounds good, I meant to ask if there was a particular reason for using `es5`. I'll make that change.